src/Refactoring-Core-Tests/ReMethodsSendNoSupersendsTest.class.st

```diff
@@ -0,0 +1,49 @@
+Class {
+	#name : 'ReMethodsSendNoSupersendsTest',
+	#superclass : 'TestCase',
+	#instVars : [
+		'model'
+	],
+	#category : 'Refactoring-Core-Tests',
+	#package : 'Refactoring-Core-Tests'
+}
+
+{ #category : 'accessing' }
+ReMethodsSendNoSupersendsTest >> model [
+
+	^ model ifNil: [
+		  model := RBNamespace onEnvironment:
+			           (RBClassEnvironment classes: {
+					            MyClassBeta.
+					            MyClassAlpha }) ]
+]
+
+{ #category : 'tests' }
+ReMethodsSendNoSupersendsTest >> testMethodNotSendingAMessageToSuper [
+	| myClassAlpha myClassBeta cond |
+	myClassAlpha := self model classNamed: #MyClassAlpha. 
+	myClassBeta := self model classNamed: #MyClassBeta.
+	
+	cond := ReMethodsSendNoSupersendsCondition new
+			class: myClassBeta
+		   targetSuperclass: myClassAlpha
+		   selectors: { #methodForPullUp }.
+		
+	" the method has a duplicate in MyClassBeta's sibling (MyClassBetaSibling), therefore the condition fails "
```

Comment:
I think this comment was left by accident @carolahp

Reply:
Thank you Balsa, I will remove it in the PR for the cleanup [issue#17158](https://github.com/pharo-project/pharo/issues/17158)